index.bs

```diff
@@ -750,15 +750,19 @@ dictionary XRReferenceSpaceOptions {
   XRStationaryReferenceSpaceSubtype subtype;
 };
 
-[SecureContext, Exposed=Window] interface XRReferenceSpace : XRSpace {
-  attribute XRRigidTransform originOffset;
+
+[SecureContext, Exposed=Window,
+ Constructor(XRReferenceSpace base, XRRigidTransform originOffset)]
```

Comment:
Can we change this pattern to not be a constructor, but rather a function on the XRReferenceSpace object being based on? Maybe `XRReferenceSpace.getOffsetReferenceSpace()`